.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+### Description
+
+<!-- What has changed and why has it changed -->
+
+#### Remaining Tasks
+
+<!-- Tasks left to do, either in this MR or as future work -->
+
+#### Related MRs in other projects
+
+<!-- Links of other related MR to this one -->
+
+#### Additional merge criteria
+
+<!-- Here, add any extra criteria you consider mandatory for merging on top of the usual criteria -->
+
+#### Breaking changes
+
+<!-- Here, add all the breaking changes that this MR involves if there is any -->
+
+### Checklist
+
+- [ ] Title of the MR follows the right format (see the internal documentation).
+
+#### Documentation
+
+- [ ] Update the version of pasqal-cloud in `_version.py` following the changes your MR implies. We use an **x**.**y**.**z** versioning, where **x** is not used yet, **y** for a breaking change and **z** for a new feature, bug correction or minor change.
```

Comment:
Good point

Reply:
`the changes your MR implies` reads a little strange.  'following the changes in your PR` perhaps.

We should reference PR not MR on Github since they call them pull requests